.pre-commit-config.yaml

```diff
@@ -15,6 +15,7 @@ repos:
   - id: requirements-txt-fixer
   - id: check-docstring-first
   - id: name-tests-test
+    args: [--django]
```

Comment:
How about putting it on a dedicated line?
```suggestion
    args:
    - --django
```

Reply:
Edited

Reply:
Marking as resolved.